util/markdownlint.rb

```diff
@@ -21,3 +21,4 @@
 exclude_rule 'MD025' # Multiple top level headers in the same document
 exclude_rule 'MD026' # Trailing punctuation in header
 exclude_rule 'MD029' # Ordered list item prefix
+exclude_rule 'MD040' # Allow fenced code blocks
```

Comment:
Please amend the comment to: `Allow fenced code blocks without a language`

Reply:
```suggestion
exclude_rule 'MD040' # Allow fenced code blocks without a language
```